ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.14.yaml

```diff
@@ -132,6 +187,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone
+- as: aws-ipi-localzone-fips-f28
```

Comment:
We already have jobs covered `cucushift-installer-rehearse-aws-ipi-edge-zone`, so don't need to add it again.